### ISSUE TITLE
Create Slack Bot button has gone missing

### ISSUE THREAD

**Issue by dacort** [metabase/metabase#12382]
**Describe the bug**
The `Create a Slack Bot User for MetaBot` button and description text is missing on the Slack settings page.
In addition, I can't click the button to disable the integration unless the token field is filled out.

The button was present in 0.34, so looks like a regression in 0.35.

Workaround:
Click on https://my.slack.com/services/new/bot
Then follow the docs here: https://www.metabase.com/docs/latest/administration-guide/09-setting-up-slack.html

Previous page:
![image](https://user-images.githubusercontent.com/1512/80044023-1d618280-84b8-11ea-8770-b339b6987194.png)

Current page:
![image](https://user-images.githubusercontent.com/1512/80044045-2baf9e80-84b8-11ea-978d-af00f3455a3c.png)

**Comment by kdoh** [comment]
Looks like the original form with that button got deleted in #11773

**Comment by paulrosenzweig** [comment]
Fixed in #12415

**Comment by ctivanovich** [comment]
This approach doesn't appear to work anymore for sending messages to Slack.

**Comment by dacort** [comment]
@ctivanovich As mentioned in the other ticket, did you try creating a bot user with the link in the workaround above? I just tried this on v0.35.2 with this link: https://my.slack.com/services/new/bot - it seems to work fine for me.

If it doesn't work for you, can you please create a new ticket with the details of your Metabase installation and steps to reproduce the issue? Thanks so much.

**Comment by ctivanovich** [comment]
@dacort Thank you for responding; I've gotten the bot working actually. I had already followed the steps above and tested it when I posted my results, but now, several hours later, the bot responds in the channel. Pulses aren't being delivered, getting a null error, but I will trouble shoot that. Thanks for following up with me! I don't know Clojure, but when I have a bit of free time I'd love to contribute to this great project if I can.
